source/developers/projects/engine/api/site/cache/statistics.rst

```diff
@@ -42,18 +42,17 @@ Response
 
 .. code-block:: json
 
-  [
-    {
-      "46b0f2effb636293234e10749dd9f5a4": {
-        "size": 147,
-        "localHeapSize": 147,
-        "localOffHeapSize": 0,
-        "localDiskSize": 0,
-        "remoteSize": 0,
-        "writerQueueLength": 0,
-      }
-    }
-  ]
+  {
+    "size": 17,
```

Comment:
I thought `size` was too expensive of a call? Is that size in number of objects or bytes?

Reply:
that is correct, the expensive one is the size in bytes